pkg/containerutil/cp_linux.go

```diff
@@ -138,14 +170,6 @@ func CopyFiles(ctx context.Context, container2host bool, pid int, dst, src strin
 		tarX = append(tarX, "--no-same-owner")
 	}
 	tarX = append(tarX, "-f", "-")
-	if rootlessutil.IsRootless() {
-		nsenter := []string{"nsenter", "-t", strconv.Itoa(pid), "-U", "--preserve-credentials", "--"}
-		if container2host {
-			tarC = append(nsenter, tarC...)
-		} else {
-			tarX = append(nsenter, tarX...)
-		}
-	}
```

Comment:
Can it be safely removed when ` status.Status == containerd.Running`?

Reply:
I don't quite understand this part, in rootless mode its already in the [namespace of the rootlesskit child pid](https://github.com/containerd/nerdctl/blob/main/pkg/rootlessutil/parent_linux.go#L66). I am missing anything here? The integration tests pass okay when `status.Status == containerd.Running` is true.